Library/Formula/ansible-cmdb.rb

```diff
@@ -16,11 +16,13 @@ class AnsibleCmdb < Formula
 
   def install
     bin.mkpath
+    man1.mkpath
     inreplace "Makefile" do |s|
       s.gsub! "/usr/local/lib/${PROG}", prefix
       s.gsub! "/usr/local/bin", bin
+      s.gsub! "/usr/local/share/man/man1/", man1 + 'ansible-cmdb.1'
```

Comment:
No, because the manual page would be installed as `ansible-cmdb.man.1`, which is wrong – no? Seems to be a bug in the upstream package, original Makefile snippet:
```
cp -a ansible-cmdb.man.1 /usr/local/share/man/man1/
```

Or do you want to install the manual page with the garbled name?